### PR TITLE
[RELAY] Fix the FoldConstant Regression for VTA

### PR DESCRIPTION
**Pull request by tqchen**
cc @tmoreau89 @anijain2305

### CODE REVIEW

**Comment by tmoreau89** [comment]
For reference here's the PR that introduced the regression. The regression was not caught because the of a buggy CI that did not catch the regression, until recently when Tianqi fixed the CI script.

**Comment by tmoreau89** [comment]
https://github.com/apache/incubator-tvm/pull/6195

**Comment by anijain2305** [comment]
Thanks @tqchen for the fix.

File: src/relay/transforms/fold_constant.cc [hunk 1]
```diff
@@ -230,7 +216,17 @@ class ConstantFolder : public ExprMutator {
     auto entry_func = Downcast<Function>(mod->Lookup("main"));
     expr = expr.as<FunctionNode>() == nullptr ? entry_func->body : entry_func;
 
-    FInterpreter executor = GetInterpreter(mod);
+    using tvm::transform::PassContext;
```
Comment (jroesch) [inline src/relay/transforms/fold_constant.cc:219]: Why inline this back in?

Comment (tqchen) [reply src/relay/transforms/fold_constant.cc:219]: We will need the make sure the context is set during the `executor(input)` call besides the creation.

**Pull request merged**